react/admin/DefaultCustomField.tsx

```diff
@@ -216,9 +216,9 @@ const DefaultCustomField: React.FC<CustomFieldProps> = ({
                     )}
                     variant="secondary"
                     onClick={() => handleDeleteDropdownItem(i)}
-                    csx={{
+                    className={csx({
```

Comment:
This was a breaking change from this version: the `csx` prop doesn't exist anymore, you need to send it within `className` instead, as explained [here](https://admin-ui.vercel.app/releases).